### DISCUSSION TITLE
Change password associated to a saved SSH connection

### DISCUSSION THREAD

**Discussion by thesudoman** [isontheline/pro.webssh.net#963] [General]
Say I have 10 saved ssh connections and I need to change the password for all 10 to something else. Is it possible to change the password for all 10 at once or do I need to change each one individually?

**Comment by isontheline** [comment]
Hello @thesudoman 👋

That's right you will need ten updates 😞

A workaround is to use a Private Key instead of password so you will be able to update only once the PKey when you update it.

But are you allowed to use the private key authentication?

**Reply by thesudoman** [reply]
Ok, no worries and thank you for the quick response. I'm not using private keys today but might transition to that in the future

**Reply by isontheline** [reply]
Thank you a lot for your feedback 🙏
